utils/utils.go

```diff
@@ -13,3 +16,23 @@ func WriteToFile(data interface{}, filePath string) error {
 	}
 	return os.WriteFile(filePath, marshaled, 0o644)
 }
+
+// RenderTemplateFile render template file
+func RenderTemplateFile(srcTemplate string, params map[string]interface{}, dest string, perm os.FileMode) error {
+	templateData, err := os.ReadFile(srcTemplate)
+	if err != nil {
+		return fmt.Errorf("error occurred while trying to read %s : %e", srcTemplate, err)
```

Comment:
not very important but you can use `errors.Wrap` or `errors.Wrapf`:
`return errors.Wrapf(err, "error occured while trying to read %s", srcTemplate)`

Reply:
I know but there was PR that removed errors from go.mod so i assumed we are using fmt for errors

Reply:
>not very important but you can use errors.Wrap or errors.Wrapf:

Why? we intentionally [got rid](https://github.com/openshift-kni/lifecycle-agent/pull/38) of the `errors` library, which is non-standard and mostly obsolete since Go 1.13 introduced a standard way to wrap errors with `%w` in `fmt.Errorf`

Having said that, the line you commented on has a typo, it should say `%w` and not `%e`

Reply:
Oh, I did not know that.

Reply:
I wonder if we can add a linter test or something else to ensure we don't inadvertently add the lib back in the future ... @jc-rh ?